wp-content/themes/core/pcss/utilities/variables/_typography.pcss

```diff
@@ -4,32 +4,38 @@
 	 * ----------------------------------------------------------------------------- */
 
 	--font-family-core-icons: 'core-icons';
-	--font-family-proxima-nova: "proxima-nova", "Helvetica Neue", Helvetica, Arial, sans-serif;
+	--font-family-calibre: "calibre", "Helvetica Neue", Helvetica, Arial, sans-serif;
+	--font-family-tiempos: "tiempos", Georgia, "Times New Roman", sans-serif;
+	--font-family-apercu: "apercu", monospace;
 	--font-family-sans-serif: "Helvetica Neue", Helvetica, Arial, sans-serif;
-	--font-family-base: var(--font-family-proxima-nova);
+	--font-family-base: var(--font-family-tiempos);
 
 	/* -----------------------------------------------------------------------------
 	 * Font Sizing
 	 * ----------------------------------------------------------------------------- */
 
-	--font-size-base: 18px;
-	--font-size-content: 18px;
-	--font-size-content-s: 16px;
+	--font-size-base: 16px;
```

Comment:
Same notes around 0 index var naming for font sizes and line-heights and example (including for the typography mixins) fwiw https://github.com/moderntribe/tribe-common-styles/tree/master/mixins